ethereum/contracts/bridge/BridgeState.sol

```diff
@@ -9,6 +9,8 @@ contract BridgeStorage {
     struct Provider {
         uint16 chainId;
         uint16 governanceChainId;
+        // Required number of block confirmations to assume finality
+        uint8 finality;
```

Comment:
does this not affect the existing state by adding this in the middle?

Reply:
see the note in the PR description (and the upgrade test)